src/Argu/Types.fs

```diff
@@ -99,6 +99,12 @@ type ArgumentType =
     /// Argument specifies a subcommand
     | SubCommand = 4
 
+type CustomAssignmentSeperator =
```

Comment:
TIL both forms are correct, but probably best to maintain consistency with what's being used in the codebase.
```suggestion
type CustomAssignmentSeparator =
```